pyproject.toml

```diff
@@ -27,6 +27,8 @@ stripe = "^10.9.0"
 types-requests = "^2.32.0.20240712"
 types-setuptools = "^71.1.0.20240813"
 python = "^3.11"
+bleach = "^6.2.0"
```

Comment:
`bleach` should be moved up to be alphabetized. `types-bleach` should be moved into dev dependencies only.

Reply:
✅